pkg/pillar/cmd/logmanager/logmanager.go

```diff
@@ -463,12 +447,11 @@ func handleDNSDelete(ctxArg interface{}, key string, statusArg interface{}) {
 
 // This runs as a separate go routine sending out data
 // Compares and drops events which have already been sent to the cloud
-func processEvents(image string, prevLastSent time.Time,
-	logChan <-chan logEntry, eveVersion string) {
-
-	log.Infof("processEvents(%s, %s)\n", image, prevLastSent.String())
+func processEvents(image string, logChan <-chan logEntry,
+	eveVersion string, ctx *logmanagerContext) {
 
 	reportLogs := new(logs.LogBundle)
+	appLogBundles := make(map[string]*logs.AppInstanceLogBundle)
```

Comment:
How about having the right had side of this be a struct so you can include the msgID?

Reply:
Will do.

Reply:
That might not work. We delete bundles that are exported to cloud. We might have to implement a separate map for maintaining per-app counters. But that brings in more complexity where we have to keep track of newly added/deleted apps.

Reply:
If we can't make the number make sense I'd prefer setting it to zero then including something which intentionally has an undeterminable amount of gaps in the sequence. Those gaps will be utterly confusing.

Reply:
setting msgId to zero for app logs for now.

Reply:
OK, once I see that code removed from the PR I'll pull this.

Reply:
It is already removed.

Reply:
Now it is; you hadn't pushed that an hour ago.

Reply:
I pushed the changes 9 hours ago. Not sure why you did not see them in PR.